packages/cli/commands/functions/list.js

```diff
@@ -36,7 +36,7 @@ exports.handler = async options => {
 
   logger.debug(i18n(`${i18nKey}.debug.gettingFunctions`));
 
-  const routesResp = await getRoutes(accountId).catch(async e => {
+  const { data: routesResp } = await getRoutes(accountId).catch(async e => {
```

Comment:
I think it may be worth unrolling the data from the response either at the `http` layer or in the `api` helpers instead of doing it where the helpers are used.